pkgs/applications/science/math/R/default.nix

```diff
@@ -49,10 +53,22 @@ stdenv.mkDerivation rec {
       LDFLAGS="-L${gfortran.cc}/lib"
       RANLIB=$(type -p ranlib)
       R_SHELL="${stdenv.shell}"
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+      --without-tcltk
+      --disable-R-framework
```

Comment:
@peti I guess, framework would work, with proper changes to wrappers, but I'm nort sure what those should be. For me it's ok to keep this open until other darwin R users are found to help with this.